compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaSymbol.java

```diff
@@ -108,6 +110,10 @@ public boolean isLangLib() {
         return isLangLib;
     }
 
+    public BSymbol getInternalSymbol() {
```

Comment:
unused method?

Reply:
In use, after the addition of the above mentioned API.